regionsize_vs_objectsize.sh

```diff
@@ -7,11 +7,11 @@ if [ -z "${log}" ] ; then
 fi
 
 total=`grep "source: concurrent humongous allocation" ${log}  | wc -l`
-fit2mb=`grep "source: concurrent humongous allocation" ${log} | sed 's/.*allocation request: \([0-9]*\) bytes.*/\1/' | awk '{if($1<1048576) print}' | wc -l`
-fit4mb=`grep "source: concurrent humongous allocation" ${log} | sed 's/.*allocation request: \([0-9]*\) bytes.*/\1/' | awk '{if($1<2097152) print}' | wc -l`
-fit8mb=`grep "source: concurrent humongous allocation" ${log} | sed 's/.*allocation request: \([0-9]*\) bytes.*/\1/' | awk '{if($1<4194304) print}' | wc -l`
-fit16mb=`grep "source: concurrent humongous allocation" ${log} | sed 's/.*allocation request: \([0-9]*\) bytes.*/\1/' | awk '{if($1<8388608) print}' | wc -l`
-fit32mb=`grep "source: concurrent humongous allocation" ${log} | sed 's/.*allocation request: \([0-9]*\) bytes.*/\1/' | awk '{if($1<16777216) print}' | wc -l`
+fit2mb=`grep "source: concurrent humongous allocation" ${log} | sed 's/.*allocation request: \([0-9]*\).*/\1/' | awk '{if($1<1048576) print}' | wc -l`
+fit4mb=`grep "source: concurrent humongous allocation" ${log} | sed 's/.*allocation request: \([0-9]*\).*/\1/' | awk '{if($1<2097152) print}' | wc -l`
+fit8mb=`grep "source: concurrent humongous allocation" ${log} | sed 's/.*allocation request: \([0-9]*\).*/\1/' | awk '{if($1<4194304) print}' | wc -l`
+fit16mb=`grep "source: concurrent humongous allocation" ${log} | sed 's/.*allocation request: \([0-9]*\).*/\1/' | awk '{if($1<8388608) print}' | wc -l`
+fit32mb=`grep "source: concurrent humongous allocation" ${log} | sed 's/.*allocation request: \([0-9]*\).*/\1/' | awk '{if($1<16777216) print}' | wc -l`
```

Comment:
There is slight difference between log lines of legacy GC options and jvm unified logging event

**Legacy**
```
 0.106: [G1Ergonomics (Concurrent Cycles) request concurrent cycle initiation, reason: occupancy higher than threshold, occupancy: 60817408 bytes, allocation request: 1048592 bytes, threshold: 60397965 bytes (45.00 %), source: concurrent humongous allocation]
```

**Unified Framework**
```
[334133.389s][debug][gc,ergo,ihop  ] Request concurrent cycle initiation (occupancy higher than threshold) occupancy: 390070272B allocation request: 524304B threshold: 350322636B (32.63) source: concurrent humongous allocation
```